README.md

```diff
@@ -45,12 +51,14 @@ Networking module is written in [**R**](https://www.r-project.org/).
 
 ## How to contribute?
 
-[Sign up](https://jira.niis.org/secure/Signup!default.jspa) for an account and
-get access to the [X-Road Service Desk](https://jira.niis.org/servicedesk/customer/portal/1) and
-[X-Road Metrics Backlog](https://jira.niis.org/projects/OPMONDEV/).
+[Sign up](https://id.atlassian.com/signup) for an account and
+get access to the
+[X-Road Metrics Backlog](https://nordic-institute.atlassian.net/jira/software/c/projects/OPMONDEV/boards/2/backlog).
 
-Submit a pull request to [X-Road Metrics source code Github repository](https://github.com/nordic-institute/X-Road-Metrics)
-or an enhancement request to the [X-Road Service Desk](https://jira.niis.org/servicedesk/customer/portal/1).
+Submit a pull request to
+[X-Road Metrics source code Github repository](https://github.com/nordic-institute/X-Road-Metrics)
+or an enhancement request to the
+[ideas category under discussions](https://github.com/nordic-institute/X-Road-Metrics/discussions/new?category=ideas).
 
 When opening a pull request, please provide a signed Contributor Licence Agreement (CLA). More information can be found [here](https://github.com/nordic-institute/X-Road-development/#contributor-licence-agreement).
```

Comment:
Let's change the CLA link so that it points to the X-Road contributing guidelines:

https://github.com/nordic-institute/X-Road/blob/develop/CONTRIBUTING.md#legal-notice